docs/UserGuide.adoc

```diff
@@ -452,21 +453,32 @@ After running a match command on a specific patient with `match ic/**NRIC**`, yo
 You can then run `sort rate` to obtain the list of donors with the highest success rate at the top of the list.  +
 
 === Processing potential matches: `processing`
-//TODO: change to shorter term that captures the meaning
 
 After finding a list of matches, the hospital is free to schedule a cross-matching test between
-a specific donor and patient. +
+a specific donor and patient. This command serves to allow you to keep track of the necessary tasks needed to do for the patient and donor pair before the cross-matching test. +
+
+The `Status` of patient and donor can be either `Not Processing`, `Processing` or `Done`. +
+All the patients and donors should be `Not Processing` initially. +
+When the command is used for the first time for the patient and donor pair, the `Status` of the patient and donor will change from `Not Processing` to `Processing`. +
+If the patient and donor is being processed, their `Status` will remain `Processing`.
```

Comment:
The paragraphing looks weird 1 sentence per line

Reply:
this is to show the processing vs not processing and vs done ulitimately